app/services/faculty_module/create_service.rb

```diff
@@ -29,7 +30,8 @@ def create
           user: user,
           category: Faculty::CATEGORY_VISITING_COACHES,
           connect_link: @faculty_params[:connect_link],
-          public: @faculty_params[:public]
+          public: @faculty_params[:public],
+          archived_at: @faculty_params[:archived_at]
```

Comment:
```suggestion
```

Reply:
I am keeping it, as we are allowing creation of archived coaches.